_config.yml

```diff
@@ -6,7 +6,7 @@ highlighter: pygments
 redcarpet:
   extensions: ["no_intra_emphasis", "fenced_code_blocks", "autolink", "strikethrough", "superscript", "with_toc_data"]
 
-baseurl: '/grey-matter/'
+baseurl: '/bcbsri/round1/'
```

Comment:
Looks great. What parts would we rollback into master? Also, I wonder if this becomes a new starting point for projects and we shift to FDD... but I could see a scenario where we don't have tons of details fleshed out but we know the vision for a site this would be great to get ramped up with.

Reply:
Right. This should get rolled back, but generally, the rest of it is in a good spot. I just finished up a vertical rhythm thing. I'll merge this stuff and then revisit what gets split out for a stand alone project.